### PR TITLE
Add option to show power stats

### LINKED ISSUE #19: Power usage
Hi,
How to add power  usage and efficiency information ?

### PR DESCRIPTION
**Pull request by cjw85**
Adds ability to show optionally power usage and limit (grouped with temperature and utilisation).

### CODE REVIEW

File: gpustat.py [hunk 1]
```diff
@@ -114,6 +114,24 @@ def utilization(self):
         return int(v) if v is not None else None
 
     @property
+    def power_usage(self):
+        """
+        Returns the GPU power usage in Watts,
+        or None if the information is not available.
+        """
+        v = self.entry['power.use']
```
Comment (wookayin) [inline gpustat.py]: Actually I think it should be "power.draw". A convention here is that I am using the same column name as shown in `nvidia-smi --help-query-gpu`.

**Comment by cjw85** [comment]
Changed: `power.use` -> `power.draw`, and `power.limit` -> `enforced.power.limit` as given by `nvidia-smi --query`. Changed API properties to be consistent (whilst not being verbose for the second case).

**Comment by wookayin** [comment]
Between `power.limit` and `enforced.power.limit`, what is the quantity you are seeking to print out? I thought it was `power.limit`, but you do `enforced.power.limit`, right?

**Comment by cjw85** [comment]
I'm calling `N.nvmlDeviceGetEnforcedPowerLimit` so `enforced.power.limit` is the correct form. This is described as "the minimum of various power limiters.", so seems to be the most relevant to display in the simple interface (and provide in the simple API).

**Comment by wookayin** [comment]
Merged, thanks!

**Pull request merged**